### PR TITLE
Added feature to scrape user bio 

### PR DESCRIPTION
**Pull request by codingis4noobs2**
Added a feature to scrape user bio

### CODE REVIEW

**Comment by nikhil25803** [comment]
Include the method you created in the table of the `documentation.md` file.

File: src/scrape_up/github/__init__.py [hunk 1]
```diff
@@ -37,3 +37,15 @@ def get_avatar(self):
         except:
             message = f"Avatart not found for username {self.username}"
             return message
+
+    def get_bio(self):
+        """
```
Comment (nikhil25803) [inline src/scrape_up/github/__init__.py:42]: Include this method in the `documentation.md` file.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/__init__.py (hunk 1 above)
Comment (codingis4noobs2) [reply src/scrape_up/github/__init__.py:42]: Sure, will do

Comment (codingis4noobs2) [reply src/scrape_up/github/__init__.py:42]: Done updating documentation.md @nikhil25803

**Review by nikhil25803 — APPROVED** [review]

**Pull request merged**